app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file, new_file = command.split()
+    if "mv" in command:
```

Comment:
```suggestion
    if "mv" == command:
```
Also check that paths old_file, new_file are not the same